packages/editor/src/components/toolbar/index.ts

```diff
@@ -43,7 +50,10 @@ export class Toolbar extends LitElement {
       value?.showModal();
     }
   };
-
+  #onToolbarFocus = () => {
```

Comment:
```suggestion
  readonly #onToolbarFocus = () => {
```

Deze is neem ik aan ook readonly?